cxx4opencl/address_spaces.txt

```diff
@@ -332,9 +332,17 @@ __global const int& f(__global float &ref) {
 
 Construction, initialization and destruction of objects in `+__private+`
 and `+__global+` address space follow the general principles of {cpp}. For
-program scope objects, the implementation (i.e. compiler) defines an ABI
```

Comment:
If I read this correctly, trivial ctor/dtor are unconditionally supported, right?

Reply:
Yes, that's right... they are resolved at compile time so to say i.e. there is nothing to be done by the runtime.